### DISCUSSION TITLE
如何区分服务器指令和MCC指令

### DISCUSSION THREAD

**Discussion by 249261550** [ORelio/Minecraft-Console-Client#2395] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/68882828/209799071-55d84570-89d5-427b-b155-1acf564ff8d2.png)

**Comment by ReinforceZwei** [comment, marked as answer]
`send` is MCC command. Everything after `send` will be sent to server. Use `Action = "inventory 1 click 1"` to execute MCC command.

**Reply by 249261550** [reply]
十分感谢  ！！！
